odoo/tools/config.py

```diff
@@ -335,10 +335,18 @@ def __init__(self, fname=None):
                              help="Maximum allowed virtual memory per worker (in bytes), when reached the worker be "
                              "reset after the current request (default 2048MiB).",
                              type="int")
+            group.add_option("--limit-memory-soft-gevent", dest="limit_memory_soft_gevent", my_default=2048 * 1024 * 1024,
```

Comment:
Looking at `db_maxconn_gevent` for comparison: the default is to stick to `db_maxconn` if `db_maxconn_gevent` is not specified. Shouldn't we use the same pattern here?
Otherwise, deployments may see a change in the memory limits applied to the gevent worker.

Reply:
You are right, updated.